disruption_generator/cli.py

```diff
@@ -31,35 +28,65 @@ def main(experiments_path):
     """Console script for disruption_generator."""
     click.echo("!!! DISRUPTION AS A SERVICE !!!")
     click.echo("!!!    USE WITH CAUTION     !!!")
+    parse_log_config(default_config_file="default_logging.yaml", custom_config_file="custom_config.yaml")
+    logger = logging.getLogger(__name__)
     try:
         loop = asyncio.get_event_loop()
-        loop.run_until_complete(execute(experiments_path))
+        loop.run_until_complete(execute(logger, experiments_path))
     except (OSError, asyncssh.Error) as exc:
         sys.exit("SSH connection failed: " + str(exc))
 
 
-async def execute(experiments_path):
+def parse_log_config(default_config_file, custom_config_file):
+    """
+    This will look at logging configuration files and load the config using native abilities of
+    python logging.config module. In the first instance, default logging config provided by
+    Disruption Generator developers is loaded. After that custom configuration provided by user is
+    loaded and it overwrites the default configuration.
+    Args:
+        default_config_file (str): Path to file hodling default configuration
+        custom_config_file (str): Path to file hodling user configuration
+    Returns:
+        None
+    """
+    try:
+        with open(default_config_file, "r") as f:
+            default_config = yaml.safe_load(f)
+            logging.config.dictConfig(default_config)
+    except FileNotFoundError:
+        sys.exit("{} not found. Logging cannot be configured.".format(default_config_file))
+    try:
+        with open(custom_config_file, "r") as f:
+            custom_config = yaml.safe_load(f)
+            logging.config.dictConfig(custom_config)
+    except FileNotFoundError:
+        # If custom config file is not found, it's no probem. Default config will be used.
+        pass
+
+
+async def execute(logger, experiments_path):
```

Comment:
Not very keen in passing the logger to the execute signature. Kind of violates the SOLID principles, particularly Single-responsibility one.
Rather have the logger as a global variable.

Reply:
Correct. Fixed it.